training/splitting_dataset.py

```diff
@@ -0,0 +1,132 @@
+import json
+from collections import defaultdict
+from typing import Dict, List, Tuple
+
+import pandas as pd
+from sklearn.model_selection import train_test_split
+
+from training.analysis_functions import DataFrameType
+
+
+_DEFAULT_TEST_SIZE: float = 0.25
+"""Default value of fraction of the dataset to include in test split."""
+
+
+def _get_book_names_from_dataframe(dataframe: DataFrameType) -> List[str]:
+    """
+    Collects unique values of "book" column in dataframe.\n
+    :param dataframe: Processed dataframe.
+    :return: List of books' names.
+    """
+    try:
+        return dataframe.book.unique()
+    except AttributeError as e:
+        print(f"_get_book_names_from_dataframe: Error occurred: {e}.")
+        return []
```

Comment:
Czy na pewno taka funkcja powinna zwracać pustą listę, jak się nie udało wydobyć książek z listy? Mam wrażenie, że jak coś takiego się nie uda, no to jest błąd krytyczny i trzeba sprawdzić, co się stało

Reply:
Nie uda się tylko w przypadku braku kolumny "book" w df.

Reply:
A brak takiej kolumny chyba nie dyskwalifikuje całkowicie zbioru z możliwości trenowania na nim, prawda?